### PR TITLE
Monorepo

### PR DESCRIPTION
**Pull request by Meyanis95**
## Motivation

This PR add to the project the SDK, that let users import prebuilt component in their app. 

## Change Summary
- Changed the structure of the project to become a monorepo using yarn workspace.
- Added SDK package "country-identity-kit"

What's not done in this PR: 
- npm publish CI

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [x] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [X] PR has been tagged with a change label(s) (i.e. documentation, feature, bug fix, or chore)
- [X] PR includes documentation if necessary.

@vuvoth WDYT?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,64 +1,29 @@
...
+    "url": "git+https://github.com/privacy-scaling-explorations/anon-aadhaar.git"
   },
-  "files": [
-    "./artifacts/verification_key.json",
-    "./src/*",
-    "./dist/*",
-    "./README.md"
+  "workspaces": [
+    "packages/country-identity-kit",
+    "packages/pcd-country-identity"
```
Comment (vuvoth) [inline package.json:21]:
```suggestion
    "packages/*",
```

File: node_modules/.yarn-integrity
[the remark this answers is not in the document]
Comment (Meyanis95) [reply node_modules/.yarn-integrity:1]: Weird!!
I'm removing it

File: packages/country-identity-kit/src/util.ts [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    const signature = Buffer.from(signatureHex, 'hex').toString('binary')
+    return {
+      ByteRange: matches.slice(1, 5).map(Number),
+      signature,
+      signedData,
+    }
+  }
+}
+
+const getSubstringIndex = (str: Buffer, substring: string, n: number) => {
```
[the remark this answers is not in the document]
Comment (Meyanis95) [reply packages/country-identity-kit/src/util.ts:98]: As of now I need it
But the pdf flow will change when it's all ready :)

File: packages/country-identity-kit/package.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    "tsconfig.json",
+    "package.json"
+  ],
+  "author": "Yanis Meziane for PSE",
+  "license": "MIT",
+  "dependencies": {
+    "@pcd/pcd-types": "^0.6.0",
+    "@peculiar/x509": "^1.9.3",
+    "node-forge": "^1.3.1",
+    "pcd-country-identity": "^0.0.5",
```
Comment (vuvoth) [inline packages/country-identity-kit/package.json:30]: I think we can use "pcd-country-indentity: workspace:*" here

**Review by vuvoth — COMMENTED** [review]
LGTM! Let a few comments.

**Review by vuvoth — APPROVED** [review]

File: packages/country-identity-kit/package.json (hunk 3 above)
Comment (Meyanis95) [reply packages/country-identity-kit/package.json:30]: I'm going to check on that!

**Pull request merged**